MuxExoPlayer/build.gradle

```diff
@@ -15,7 +15,7 @@ android {
     }
     buildTypes {
         release {
-            minifyEnabled true
+            minifyEnabled false
```

Comment:
Does this not work if we minify the builds any more? Is this the part that fixes the issue?

Reply:
I think we can minify, it is not part of the issue, I will doublecheck now.